backend/api/admin/src/main/resources/application.properties

```diff
@@ -3,3 +3,6 @@ kafka.schema-registry-url=${KAFKA_SCHEMA_REGISTRY_URL}
 kafka.cleanup=${KAFKA_CLEANUP:false}
 kafka.commit-interval-ms=${KAFKA_COMMIT_INTERVAL_MS}
 kubernetes.namespace=${KUBERNETES_NAMESPACE}
+segment.analytics.enabled=${TRACKING_ENABLED}
```

Comment:
No need to put this in the `application.properties` explicitly we have found. You can simply directly reference the variable in the `@Value` annotation.